presets/minimal/whisk.yaml

```diff
@@ -0,0 +1,20 @@
+# Minimal preset - Whisk
+
+# Misc
+# ---------------------------------------------------------------
+# [customized]
+CURDLEPROOFS_N_BLINDERS: 2
```

Comment:
Curdleproofs always requires 4 blinders with the current state of the protocol.

Reply:
Noting that https://github.com/nalinbhardwaj/curdleproofs.pie is able to validate a generated proof with those parameters